graphql/e2e/custom_logic/cmd/main.go

```diff
@@ -134,7 +134,11 @@ func check2(v interface{}, err error) {
 }
 
 func getError(key, val string) error {
-	return fmt.Errorf(`{ "errors": [{"message": "%s: %s"}] }`, key, val)
```

Comment:
`strconv.Quote()` would not work for this use case.
See: https://play.golang.org/p/JOton9hhrww

The reason is, inside a JSON string any double quote should be escaped, `strconv.Quote(arg)` would escape the double quotes inside the `arg` but it would again put double quotes at the beginning and end of the `arg`,